tailscale/DOCS.md

```diff
@@ -17,8 +17,55 @@ the following URL:
 
 <https://login.tailscale.com/start>
 
-You can also create an account during the add-on installation processes,
-however, it is nice to know where you need to go later on.
+<details>
+<summary>
+<i>Additional prerequisites if you want to access your Home Assistant device
+with a Tailscale provided certificate, ie. through the Tailscale Proxy feature
+<b>(click to open)</b></i>
+</summary>
+
+### Home Assistant configuration for Tailscale Proxy feature
+
+You must configure Home Assistant to **not** use SSL certificates, to be
+accessible through plain http connection. The Tailscale https Proxy will access
+Home Assistant through `localhost` and will not accept a real certificate,
+connection will be closed with `proxy error: x509: cannot validate certificate
+for 127.0.0.1 because it doesn't contain any IP SANs`
+
+If you still want to use another https connection to access Home Assistant
+through another network, please use the **NGINX Home Assistant SSL proxy**
+add-on.
+
+So these lines have to be removed from your `/config/configuration.yaml`:
+
+```yaml
+http:
+#  ssl_certificate: /ssl/fullchain.pem
+#  ssl_key: /ssl/privkey.pem
+```
+
+Since Home Assistant by default blocks requests from proxies/reverse proxies,
+you need to tell your instance to allow requests from the Tailscale add-on.
+In order to do so, add the following lines to your `/config/configuration.yaml`
+without changing anything:
```

Comment:
```suggestion
Since Home Assistant, by default, blocks requests from proxies/reverse proxies,
you need to tell your instance to allow requests from the Tailscale add-on.
In order to do so, add the following lines to your `/config/configuration.yaml`
without changing anything:
```